### ISSUE TITLE
`reacord.reply` for non-interaction events

### ISSUE THREAD

**Issue by domin-mnd** [itsMapleLeaf/reacord#38]
Hello, I was wondering if there is any internal support for `message.reply()` (not the `interaction.reply()`) in discord.js adapter. I spent some time exploring reacord's codebase and so far I've found out that a `send` method may support options or may be somewhat rewritten to expose adapter's internals to make changes:
https://github.com/itsMapleLeaf/reacord/blob/82b3575f2d1e0699c73a03b394943f2f04eec6a5/packages/reacord/library/core/reacord-discord-js.ts#L105

The question is: can I use `message.reply()` instead of `message.send()` for message event?

**Comment by domin-mnd** [comment]
Do we need `ephemeralReply` as a method in adapter? Ephemeral state may become an option in `reply` method. Other than that, should these methods be explicitly set in adapter as was mentioned in `Reacord` class?
https://github.com/itsMapleLeaf/reacord/blob/82b3575f2d1e0699c73a03b394943f2f04eec6a5/packages/reacord/library/core/reacord.tsx#L26-L28

I'd like to open a pull request but I'm not sure what options argument should have besides the `asReply` or `reply` option.

**Comment by itsMapleLeaf** [comment]
`ephemeralReply` existed initially because normal replies and ephemeral replies had a lot of differences, but I don't think that's true anymore, so we should probably deprecate it tbh

```ts
  /** 
   * @deprecated Use reacord.reply(content, { ephemeralReply: true })
   */
  abstract ephemeralReply(...args: unknown[]): ReacordInstance
```

**Comment by domin-mnd** [comment]
Resolved, see #39
